src/V3Randomize.cpp

```diff
@@ -1649,23 +1649,27 @@ class RandomizeVisitor final : public VNVisitor {
         }
         // MemberSel: compare object and member (obj.y)
         if (VN_IS(argExpr, MemberSel) && VN_IS(withExpr, MemberSel)) {
-            const AstNode* withObj = withExpr->op1p();
-            const AstNode* argObj = argExpr->op1p();
+            const AstMemberSel* const withMS = VN_AS(withExpr, MemberSel);
+            const AstMemberSel* const argMS = VN_AS(argExpr, MemberSel);
+            if (withMS->varp() != argMS->varp()) return false;
+            const AstNode* withObj = withMS->fromp();
+            const AstNode* argObj = argMS->fromp();
```

Comment:
Use p suffix on pointers.  I'd just inline the ->fromp()'s.  

```suggestion
            const AstMemberSel* const withMSp = VN_AS(withExpr, MemberSel);
            const AstMemberSel* const argMSp = VN_AS(argExpr, MemberSel);
            if (withMS->varp() != argMSp->varp()) return false;
```